rotkehlchen/assets/utils.py

```diff
@@ -74,10 +75,19 @@ def get_or_create_evm_token(
     )
     with userdb.get_or_create_evm_token_lock:
         try:
-            ethereum_token = EvmToken(
-                identifier=identifier,
-                form_with_incomplete_data=form_with_incomplete_data,
-            )
+            ethereum_token = EvmToken(identifier=identifier)
```

Comment:
I don't understand what you are doing here. Can you explain? Best in discord.

Also form_with_incomplete_data should be explained in the docstring

Reply:
we discussed in discord. Changes were made